UPGRADE-4.0.md

```diff
@@ -913,7 +913,9 @@ Workflow
 --------
 
  * Removed class name support in `WorkflowRegistry::add()` as second parameter.
-
+ * Deprecated the `add` method in favor of the `addWorkflow` method in `Workflow\Registry`
+ * Deprecated the interface `SupportStrategyInterface` in favor of the interface `SupportStrategyWorkflowInterface`
+  
```

Comment:
And now there are two spaces here. You need to configure your editor to remove automatically all trailing spaces :D

Reply:
Yeah, you are right, I didn't configure it to do so !